http_test.go

```diff
@@ -374,3 +375,30 @@ func testServerImportHelper(t *testing.T, data interface{}) {
 
 	assert.Equal(t, http.StatusBadRequest, w.Code, "Test server returned wrong HTTP response code")
 }
+
+func BenchmarkNewSortableJSONMetrics(b *testing.B) {
+	const numWorkers = 100
+	filename := filepath.Join("fixtures", "import.deflate")
+	contentEncoding := "deflate"
+
+	f, err := os.Open(filename)
+	assert.NoError(b, err, "Error reading response fixture")
+	defer f.Close()
+
+	r := httptest.NewRequest(http.MethodPost, "/import", f)
+	r.Header.Set("Content-Encoding", contentEncoding)
+
+	w := httptest.NewRecorder()
+
+	config := localConfig()
+	s := setupVeneurServer(b, config, nil, nil, nil)
```

Comment:
It seems like this server is only being used for its trace client.  Could you maybe remove this and use `trace.DefaultClient` instead when unmarshalling the metrics?